README.md

```diff
@@ -5,18 +5,22 @@ This is the official operator to deploy and manage IBM block storage CSI driver.
 
 Supported container platforms:
   - OpenShift v4.2
+  - OpenShift v4.3
   - Kubernetes v1.14
+  - Kubernetes v1.16
 
 Supported IBM storage systems:
   - IBM FlashSystem 9100
-  - IBM Spectrum Virtualize
-  - IBM Storwize
+  - IBM Spectrum Virtualize Family (including IBM Flash family members built with IBM Spectrum Virtualize (FlashSystem 5010, 5030, 5100, 7200, 9100, 9200, 9200R) and IBM SAN Volume Controller (SVC) models SV2, SA2)
   - IBM FlashSystem A9000/R
+  - IBM DS8880
```

Comment:
Instead of these two lines, according to the latest RN - please have this as one line stating "IBM DS8000 Family"